README.md

```diff
@@ -45,10 +45,23 @@ Fun Bus wants you to make their site more interactive. They are relying on you t
 	* [ ] `select`
 	* [ ] `dblclick`
 
+
+	<!--*** MY ORDER ***-->
+	* [ ] `mouseover`
+	* [ ] `mouseout`
+	* [ ] `click w/preventDefault() and stopPropagation()`
+	* [ ] `wheel`
+	* [ ] `dblclick`
+	* [ ] `click`
+	* [ ] `copy`
+	* [ ] `(another) click`
+	* [ ] `keydown`
+	* [ ] `keyup`
```

Comment:
Fun fact - Placing an x instead of a space between these brackets changes the way they look. Try typing
```
* [x] `mouseover`
* [x] `mouseout`
* [x] `click w/preventDefault() and stopPropagation()`
```
And so on, and watch those checkboxes get marked off!